compiler/internal/analysis/info.go

```diff
@@ -393,25 +459,27 @@ func (fi *FuncInfo) Visit(node ast.Node) ast.Visitor {
 	// needs to be analyzed.
 }
 
-func (fi *FuncInfo) visitCallExpr(n *ast.CallExpr) ast.Visitor {
+func (fi *FuncInfo) visitCallExpr(n *ast.CallExpr, forDefer bool) ast.Visitor {
```

Comment:
nit: it's a personal preference, but I think `deferredCall bool` would be a bit more descriptive.